Source/WebCore/rendering/RenderQuote.cpp

```diff
@@ -99,28 +98,36 @@ struct SubtagComparison {
     int comparison;
 };
 
-static SubtagComparison subtagCompare(const char* key, const char* range)
+static int strcmpSpan(std::span<const LChar> s1, std::span<const LChar> s2)
+{
+    int result = memcmp(s1.data(), s2.data(), std::min(s1.size(), s2.size()));
+    if (!result && s1.size() != s2.size())
+        result = (s1.size() > s2.size()) ? 1 : -1;
+    return result;
+}
```

Comment:
I suppose we can move this into a header later, when we find somewhere else we need it.